cmd/suggestion/bayesianoptimization/v1alpha2/Dockerfile

```diff
@@ -2,7 +2,7 @@ FROM python:3
 
 ADD . /usr/src/app/github.com/kubeflow/katib
 WORKDIR /usr/src/app/github.com/kubeflow/katib/cmd/suggestion/bayesianoptimization/v1alpha2
-RUN if [ "$(uname -m)" = "ppc64le" ]; then \
+RUN if [ "$(uname -m)" = "ppc64le" ] || [ "$(uname -m)" = "aarch64" ]; then \
```

Comment:
I think we do not need to update v1alpha2 since it is deprecated. Or you have to replace python:3 with python:3.6 in all suggestion images. Python 3.8 is released and python:3 is redirected to python:3.8 . But it has some problems when pip install.

Reply:
@mrxinwang

We aleady set python:3 to python:3.6, you just need another rebase now.

Reply:
@gaocegege Thanks very much for your kindly suggestion. I have changed all base image to 3.6. Also, should I keep all python in these images as version 3.6? I am now using 3.7 in narsl/v1alpha3/Dockerfile.aarch64 because tensorflow 1.14.0 doesnt officially support arm64. If it is needed then I change

Reply:
No need to do it. 3.7 LGTM if it works for you.